OWNERS

```diff
@@ -52,7 +31,6 @@ reviewers:
   - paveldournov
   - qimingj
   - rileyjbauer
-  - thedriftofwords
   - vicaire
```

Comment:
We can remove the Approvers from the Reviewers list - ultimately when it comes to approval, you are forced to approve. Also I think reviewers list is quite long - and its meant for automatically rerouting reviewing requests. I believe some folks like 'paveldournov' etc would not want to be assigned review requests until they are asked explictly? So you would have a better way of knowing who are folks actively reviewing here?

Reply:
Other way is to go with a longer list, and thin down in future based on frequency of folks reviewing

Reply:
Agreed that it'd be nice to remove the approvers from the reviewers list, to make the list less overwhelming. @abhi-g do you know: does an approver have the ability to `/lgtm` if they're not listed as a reviewer?

A reason for having people in the reviewers list is to relieve the burden on the approvers - if a reviewer gives an LGTM for their area of the docs, then the approver doesn't need to do a full review. A quick sanity check is usually enough.

Reply:
I've removed the approvers from the reviewers list.